android/settings.gradle

```diff
@@ -0,0 +1,4 @@
+rootProject.name = 'native-store'
+
+include ':android'
+include ':example'
```

Comment:
it was added when I was looking for a solution launch unit test from native-store/android . I checked it, and it is unnecessary. I'll remove it.